.gitignore

```diff
@@ -1,2 +1,16 @@
 *~
```

Comment:
Do not know if we really need this file. My opinion is that all the ignores shall be local to each user. I mean per-user .gitignore instead of per-repository.

Reply:
Helps me, but I'm happy to remove it from the commit if you want.